src/DataStructures.jl

```diff
@@ -113,4 +113,6 @@ module DataStructures
 
     include("dibit_vector.jl")
     include("deprecations.jl")
+
+    include("red_black_tree.jl")
```

Comment:
I feel like deprecations being last is a good idea?
```suggestion
        include("red_black_tree.jl")
        
        include("deprecations.jl")
```